### PR TITLE
Change authors list style to bullet point list

### PR DESCRIPTION
**Pull request by Kiisu-Master**
Actually i dont know if this is the correct way of doing lists.
But it seems to work in this case.

### CODE REVIEW

File: src/ui_parts/about_menu.gd [hunk 1]
```diff
@@ -14,14 +14,10 @@ func add_section(section_title: String, authors: Array[String]):
 	authors_label.add_text(section_title + ":")
 	authors_label.pop()
 	authors_label.newline()
-	authors_label.push_table(1)
 	for author in authors:
-		authors_label.push_cell()
-		authors_label.push_bgcolor(Color(0.15, 0.2, 0.3))
+		authors_label.push_list(0,RichTextLabel.LIST_DOTS, false)
```
Comment (Kiisu-Master) [inline src/ui_parts/about_menu.gd:18]:
```suggestion
		authors_label.push_list(0, RichTextLabel.LIST_DOTS, false)
```

**Comment by Kiisu-Master** [comment]
Ok turns out this is not the correct way of doing lists.

**Comment by Kiisu-Master** [comment]
Ok turns out doing it the "correct way " is more complicated

**Comment by Kiisu-Master** [comment]
Nevermind

**Comment by Kiisu-Master** [comment]
It's ready

**Comment by MewPurPur** [comment]
Thanks!

**Pull request merged**